index.html

```diff
@@ -104,11 +115,25 @@ <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
             </ul>
             <button type="button" class="see-projects">See Projects</button>
          </article>
+         <article class="article-card">
+            <img src="./images/Imgplaholder.svg" alt="Image placeholder">
+            <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
```

Comment:
- [ ] Please use the correct `font-family` as specified in the Figma design for your project titles. Do this for each of your `recent-work__items` classes. 🙏

Reply:
Good 👍 implementing the changes